Frameworks/UIKit/UIImage.mm

```diff
@@ -382,7 +382,7 @@ - (instancetype)initWithContentsOfFile:(NSString*)pathAddr {
 
         if (EbrAccess(pathStr, 0) == -1) {
             id pathFind =
-                [bundle pathForResource:[NSString stringWithCString:newStr] ofType:nil inDirectory:nil forLocalization:@"English"];
+                [bundle pathForResource:[NSString stringWithCString:newStr] ofType:@"png" inDirectory:nil forLocalization:@"English"];
```

Comment:
Yeah anytime we are hardcoding a file extension, I'm a little dubious if that is the correct approach.

---
In reply to: [78836323](https://github.com/Microsoft/WinObjC/pull/1004#discussion_r78836323) [](ancestors = 78836323)